README.md

```diff
@@ -50,13 +50,15 @@ from your Azure app:
 ```python
 AZURE_AUTH = {
     "CLIENT_ID": "<client id>",
-    "CLIENT_SECRET": "<client secret>",
+    "CLIENT_TYPE": "confidential", # Optional, pick "public" or "confidential" (default)
```

Comment:
The terminology is "public client" and "confidential client" as per https://oauth.net/2/client-types/

Reply:
Thanks - I've update the names to follow the spec.